example/user/user.override.go

```diff
@@ -0,0 +1,7 @@
+package user
+
+import "time"
+
+func (m *UserORM) AfterToPB(user *User) {
```

Comment:
Add documentation to demonstrate what this does, so that if someone were to read this code they'd know it's a means to supply additional customization.

The hope here is that if someone comes to us and asks, "how do I add synthetic fields?" we can point them to this file and they'd understand.